tools/cloudflow-sbt-plugin/src/main/scala/cloudflow/sbt/CloudflowKeys.scala

```diff
@@ -82,6 +82,7 @@ trait CloudflowTaskKeys {
   val extraDockerInstructions = taskKey[Seq[sbtdocker.Instruction]]("A list of instructions to add to the dockerfile.")
   val verifyBlueprint = taskKey[Unit]("Verify Blueprint.")
   val printAppGraph = taskKey[Unit]("Print graph of all streamlets and how they are connected.")
+  val saveAppGraph = taskKey[Unit]("Save graph of all streamlets and how they are connected as a text file.")
```

Comment:
can it be `taskKey[File]` since it's the output of it?

Reply:
Oh, you are definately right!

Reply:
Done!